pages/content/amp-dev/community/governance.md

```diff
@@ -49,7 +49,7 @@ The Advisory Committee provides perspective and advice to the Technical Steering
 
 * The TSC's primary role is setting AMP's technical & product direction based on the [project guidelines](https://www.ampproject.org/about/amp-design-principles/).
 * Creates a product roadmap in consultation with the Working Groups.
-* Creates Working Groups and sets the initial membership & Facilitator of the Working Groups.  The TSC may initiate the creation of Working Groups or a group of people with a common interest may request recognition as a Working Group.
+* Creates Working Groups and sets the initial membership & initial Facilitator of the Working Groups.  The TSC may initiate the creation of Working Groups or a group of people with a common interest may request recognition as a Working Group.
```

Comment:
where do these changes come from? Are they because of updated imports?

Reply:
Reset this file because changes came from an imported file